File: src/main/java/com/wgzhao/addax/admin/dto/VwImpEtlListDto.java
package com.wgzhao.addax.admin.dto;

public record VwImpEtlListDto(
        String tid,
        String destOwner,
        String sysName,
        String souOwner,
        String destTablename,
        String flag,
        int retryCnt,
        int runtime,
        String filterColumn
) {
}

File: src/main/kotlin/com/wgzhao/addax/admin/dto/VwImpEtlListDto.kt
package com.wgzhao.addax.admin.dto

@JvmRecord
data class VwImpEtlListDto(
    val tid: String?,
    val destOwner: String?,
    val sysName: String?,
    val souOwner: String?,
    val destTablename: String?,
    val flag: String?,
    val retryCnt: Int,
    val runtime: Int,
    val filterColumn: String?
)
